esp-hal/src/pcnt/mod.rs

```diff
@@ -1,40 +1,24 @@
-//! # Pulse Counter peripheral driver
+//! # Pulse Counter (PCNT)
 //!
 //! ## Overview
-//! The `PCNT (Pulse Counter)` driver for `ESP` chips is a software component
-//! that provides an interface for controlling and utilizing the `PCNT`
-//! peripheral. The `PCNT` peripheral is a hardware module available in `ESP`
-//! chips, which functions as a pulse counter and encoder. It is capable of
-//! counting pulses and monitoring changes in signal levels from external
-//! sources.
+//! The PCNT module is designed to count the number of rising
+//! and/or falling edges of input signals. The may contain multiple pulse
```

Comment:
```suggestion
//! and/or falling edges of input signals. They may contain multiple pulse
```
Did you mean something like this?